{"name": "user tries to log in with different failing credentials -- @1.1 ", "status": "passed", "steps": [{"name": "Given I go to start page", "status": "passed", "start": 1609014933135, "stop": 1609014940224}, {"name": "When I click log in button", "status": "passed", "start": 1609014940224, "stop": 1609014942006}, {"name": "When I put my email \"FakeUser\" and password \"<password>\"", "status": "passed", "start": 1609014942006, "stop": 1609014942163}, {"name": "And I confirm the log in", "status": "passed", "start": 1609014942163, "stop": 1609014944222}, {"name": "Then wrong email or password shows up", "status": "passed", "start": 1609014944222, "stop": 1609014944250}, {"name": "And close browser", "status": "passed", "start": 1609014944250, "stop": 1609014944300}], "parameters": [{"name": "username", "value": "FakeUser"}, {"name": "passwor", "value": "FakePasswor"}], "start": 1609014933133, "stop": 1609014944300, "uuid": "e003b838-cd73-424c-934e-f504ab083cbb", "historyId": "e508437f2f7dc6a46b1e0650052843ed", "fullName": "features/login.feature:user tries to log in with different failing credentials", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "User logs"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}